orchagent/sainet.h

```diff
@@ -0,0 +1,105 @@
+// Header file definine base types and simple helper functions for network in SAI
+// Should keep the dependency as minimal as possible
+//
+#pragma once
+
+#include <assert.h>
+#include <memory.h>
+#include <net/if.h>
+extern "C" {
+#include "sai.h"
+}
+#include "ipaddress.h"
+#include "ipprefix.h"
+
+inline static sai_ip_address_t& copy(sai_ip_address_t& dst, const swss::ip_addr_t& src)
```

Comment:
are we using any ip_addr_t in our code? just move the content of this function into the copy(sai_ip_address_t, IpAddress). We shan't use ip_addr_t structure in our code except here in this file.